app/assets/javascripts/admin_search.js

```diff
@@ -1,48 +1,4 @@
 $(document).on('turbolinks:load', function () {
-  // Used to toggle the visibility of the advanced search filters on friend and user admin search
-  $('[data-toggle-search-filters=true]').on('click', function () {
-    $('.search-filters').toggle();
-    return false;
-  });
-
-  // Used on volunteer admin search, selecting a volunteer type
-  $('#volunteer_type').on('change', function () {
-    $(this).closest('form').submit()
-  });
-
-  // For friend advanced search, submit search form (since the submit on enter doesn't work with multiple text fields in a form)
-  $('#friend-query').on('keyup', function(e) {
-    if (e.which == 13) {
-      this.closest('form').submit()
-    }
-  });
-
-  // Submit the friend search form if there are changes and the deadline range is valid
-  $('[data-search-filter=true]').on('change', function (e) {
-    if (validateDateRange('deadlines_ending') && validateDateRange('created_at')) {
-      $(this).closest('form').submit();
-    }
-  });
-
-  // For friend search, initialize datepickers for the deadlines
+  // For admin/friend and admin/user search, initialize datepickers for the deadlines
   $(".datepicker").datepicker();
 });
```

Comment:
Do we want to include this every via the asset pipeline now? We could just add these to the individual pages. Let me know what you think.